### DISCUSSION TITLE
Markdown links to category index Pages

### DISCUSSION THREAD

**Discussion by luckyluggi** [facebook/docusaurus#8687] [Q&A]
Hello. i Have the following structure:
```
- category_1
- category_2
- mycategory
- -index.md
- -test.md
- -hello.md
```

The test.md file looks like this:
``` markdown
- [Go to category index](./index.mdx)
- [go to hello page](./second.mdx)
```

This does generate this html:
``` html
<ul class="">
    <li>
        <a href="/test/index.mdx">Go to category index</a>
    </li>
    <li>
        <a href="/test/second.mdx">go to hello page</a>
    </li>
</ul>
```

The html has 2 problems:
1. I need to remove the `.mdx` file extension to make it work. (I can already do that in the markdown and it works, but it would be great to have docusaurus just remove the extension, since vscode automatically inserts it)
2. The link to the index file is still `/test/index` which does not work since docusaurus created the rout `/test` for this page.

Is there a setting for those 2 problems or should i file an issue?

**Comment by slorber** [comment]
I don't understand your problem, please create a repro and try to explain better with runnable code that we can inspect

> I need to remove the .mdx file extension to make it work

Why do you need to remove/change the extension? Please explain

---

What I see is that you use `.md` in your file extensions, and `.mdx` in your links: that seems weird to me.
